docs/release_guide.md

```diff
@@ -145,7 +157,7 @@ Make an announcement on the [OSM mailing list](https://groups.google.com/g/opens
 
 Skip this step if the release is a release candidate (RC).
 
-Open a pull request against the `main` branch making the same version updates as [above](#update-release-branch-with-patches-and-versioning-changes) so the latest release assets are referenced there.
+Open a pull request against the `main` branch to update the `version` field in `charts/osm/Chart.yaml` to the release version.
```

Comment:
I'm guessing we can't just keep the chart version as `latest-main` because `helm` needs it to be a valid semantic version?